### ISSUE TITLE
Suggests badges not working because of cross-origin browser policies

### ISSUE THREAD

**Issue by lukeramsden** [badges/shields#851]
If you try to suggest badges it throws up this error:
```
XMLHttpRequest cannot load https://img.shields.io/$suggest/v1?url=https%3A%2F%2Fgithub.com%2FTissLTD%2Ftiss_mobile. The 'Access-Control-Allow-Origin' header has a value 'http://shields.io' that is not equal to the supplied origin. Origin 'https://shields.io' is therefore not allowed access.
```

**Comment by paulmelnikow** [comment]
Hi, thanks for reporting this. I can't reproduce this problem. I went to https://shields.io/, entered `https://github.com/TissLTD/tiss_mobile`, and clicked Suggest badges.

This was the JSON response:

```json
{"badges":[],"err":{}}
```

Where were you loading the page from?

**Comment by paulmelnikow** [comment]
This was fixed in 9816a50. Dupe of #779.
